common/data_refinery_common/models/new_models.py

```diff
@@ -0,0 +1,349 @@
+    source_url = models.CharField(max_length=256)
+
+    # Properties
+    title = models.CharField(max_length=256)
+    description = models.TextField()
+    platform_accession_code = models.CharField(max_length=256)
+    platform_name = models.CharField(max_length=256)
+
+    # Common Properties
+    is_public = models.BooleanField(default=False)
+    created_at = models.DateTimeField(editable=False, default=timezone.now)
+    last_modified = models.DateTimeField(default=timezone.now)
+
+class ExperimentAnnotation(models.Model):
+    """ Non-standard information associated with an Experiment """
+
+    class Meta:
+        db_table = "experiment_annotations"
+
+    # Relations
+    experiment = models.ForeignKey(Experiment, blank=False, null=False, on_delete=models.CASCADE)
+
+    # Properties
+    key = models.CharField(max_length=255)
+    value = models.CharField(max_length=255)
+
+    # Common Properties
+    is_public = models.BooleanField(default=False)
+    created_at = models.DateTimeField(editable=False, default=timezone.now)
+    last_modified = models.DateTimeField(default=timezone.now)
+
+class ComputationalResult(models.Model):
+    """ Meta-information about the output of a computer process. (Ex Salmon) """
+
+    class Meta:
+        db_table = "computational_results"
+
+    def __str__ (self):
+        return "ComputationalResult: " + str(self.pk)
+
+    command_executed = models.CharField(max_length=255, blank=True)
+    program_version = models.CharField(max_length=255)
+    system_version = models.CharField(max_length=255) # Generally defined in from data_refinery_workers._version import __version__
+    is_ccdl = models.BooleanField(default=True)
+
+    # Stats
+    time_start = models.DateTimeField(blank=True, null=True) 
+    time_end = models.DateTimeField(blank=True, null=True)
+
+    # Common Properties
+    is_public = models.BooleanField(default=False)
+    created_at = models.DateTimeField(editable=False, default=timezone.now)
+    last_modified = models.DateTimeField(default=timezone.now) 
+
+class CompultationalResultAnnotation(models.Model):
+    """ Non-standard information associated with an ComputationalResult """
+
+    class Meta:
+        db_table = "computational_result_annotations"
+
+    # Relations
+    result = models.ForeignKey(ComputationalResult, blank=False, null=False, on_delete=models.CASCADE)
+
+    # Properties
+    data = HStoreField(default={})
+
+    # Common Properties
+    is_public = models.BooleanField(default=False)
+    created_at = models.DateTimeField(editable=False, default=timezone.now)
+    last_modified = models.DateTimeField(default=timezone.now) 
+
+# TODO
+# class Gene(models.Model):
+    """ A representation of a Gene """
+
+#     class Meta:
+#         db_table = "genes"
+
+class Index(models.Model):
+    """ A special type of process result, necessary for processing other SRA samples """
+
+    class Meta:
+        db_table = "index"
```

Comment:
TranscriptIndex doesn't seem too bad. I've used TranscriptomeIndex a bunch of places, but that might be a bit too specific for this table. @jaclyn-taroni, would either of those be accurate for the various kinds of indices the index refinery will create? I believe that we're going to store STAR Indices in this table as well. Is that correct @miserlou?

Reply:
This would store both transcriptome index generated with Salmon and genome index generated with STAR is that correct / the meaning behind "multiple types" @kurtwheeler @Miserlou ?

Reply:
I believe that is the plan for this table but @Miserlou can confirm. Would GeneticIndex be broad enough to encompass both of those?

Reply:
I think so? Yes, that is what I meant by multiple types. I don't know anything about STAR yet.

Reply:
I _believe_ each tool of this "flavor" will require its own index to be built. What about calling this `OrganismIndex` and then the type for each tool something more specific like `SalmonIndex` and `STARIndex`?

Reply:
The way that these objects are used, I think it makes more sense just to have the table as OrganismIndex and then we can have a `program` field for `SALMON`, `STAR`, etc. rather than separate tables. But yeah.

Reply:
Does that work for you, @kurtwheeler ?

Reply:
Yup, that sounds great.